internal/index/indexscanner/scanner.go

```diff
@@ -108,6 +108,9 @@ func ReadPlugin(f io.ReadCloser) (index.Plugin, error) {
 func ReadReceiptFromFile(path string) (index.Receipt, error) {
 	var receipt index.Receipt
 	err := readFromFile(path, &receipt)
+	if receipt.Status.Source.Name == "" {
+		receipt.Status.Source.Name = constants.DefaultIndexName
```

Comment:
Aren't you missing a test now? I think there should be one where there's no status in the original manifest, but when you read the receipt, you should check that the default index was filled in.

Reply:
Ah you're right, my bad. I've been changing this a lot and missed that. I can add it back in.